src/api.ts

```diff
@@ -7,6 +7,8 @@ import * as compose from 'resin-compose-parse';
 import * as models from './models';
 import { Dict } from './types';
 
+const MAX_CONCURRENT_REQUESTS = 5;
```

Comment:
Maybe we could also take this from the env?

Reply:
I think I'll leave this as is for now as I can't think of a convincing use-case. It sort of defeats the purpose if someone can pass `MAX_CONCURRENT_REQUESTS=1000`, right? :)